nominatim/tools/refresh.py

```diff
@@ -146,6 +148,33 @@ def import_wikipedia_articles(dsn: str, data_path: Path, ignore_errors: bool = F
 
     return 0
 
+def import_osm_views_geotiff(conn: Connection, data_path: Path) -> int:
+    """ Replaces the OSM views table with new data.
+
+        Returns 0 if all was well and 1 if the OSM views GeoTIFF file could not
+        be found. Throws an exception if there was an error reading the file.
+    """
+    datafile = data_path / 'osmviews.tiff'
+
+    if not datafile.exists():
+        return 1
+
+    with conn.cursor() as cur:
+        _require_version('PostgreSQL server',
+                         conn.server_version_tuple(),
+                         POSTGRESQL_REQUIRED_VERSION)
+        _require_version('PostGIS',
+                         conn.postgis_version_tuple(),
+                         POSTGIS_REQUIRED_VERSION)
```

Comment:
You don't need to do the version checks against the required versions again, it has been done on database import. However, as the postgis_raster extension only exists for postgis >= 3.0, you should check for that and return an appropriate error if the version is smaller. The result should be that on import the loading of the tiff is simply skipped and on refresh there is an error.